File: module_app/src/main/java/com/fly/tour/MyApplication.java
package com.fly.tour;

import com.fly.tour.api.RetrofitManager;
import com.fly.tour.common.BaseApplication;

/**
 * Description: <MyApplication><br>
 * Author:      gxl<br>
 * Date:        2018/12/27<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
public class MyApplication extends BaseApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        //NewsDBManager.getInstance(this).initNewsDB();
        RetrofitManager.init(this);
    }
}

File: module_app/src/main/java/com/fly/tour/MyApplication.kt
package com.fly.tour

import com.fly.tour.common.BaseApplication
import com.fly.tour.common.manager.NewsDBManager

/**
 * Description: <MyApplication><br>
 * Author:      mxdl<br>
 * Date:        2020/2/16<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
class MyApplication : BaseApplication() {
    override fun onCreate() {
        super.onCreate()
        NewsDBManager.getInstance(this)?.initNewsDB()
    }
}